compose/ui/ui/src/uikitMain/kotlin/androidx/compose/ui/scene/ComposeSceneMediator.uikit.kt

```diff
@@ -309,17 +302,15 @@ internal class ComposeSceneMediator(
             }
         )
     }
-    @OptIn(ExperimentalComposeApi::class)
+
     private val keyboardManager by lazy {
         ComposeSceneKeyboardOffsetManager(
-            view = viewForKeyboardOffsetTransform,
+            view = view,
             keyboardOverlapHeightChanged = { height ->
-                if (configuration.platformLayers) {
-                    if (keyboardOverlapHeight != height) {
-                        animateKeyboardOffsetChanges = false
-                    }
+                if (keyboardOverlapHeight != height) {
+                    animateKeyboardOffsetChanges = false
+                    keyboardOverlapHeight = height
```

Comment:
Previously it wasn't under the condition. Looks suspicious. Let's avoid introducing unrelated changes.

Reply:
Disagree. It is code optimisation.